pom.xml

```diff
@@ -84,6 +84,7 @@
     <junit.version>4.12</junit.version>
     <slf4j.version>1.7.5</slf4j.version>
     <spark.version>2.3.1</spark.version>
+    <google.core.version>1.93.3</google.core.version>
```

Comment:
Will this affect when use in combination with other google-cloud connectors ?

Reply:
Oh good catch, these changes were added as part of testing. They didnt affect anything so I'll revert this file